doc/about.md

```diff
@@ -1,8 +1,8 @@
-The OpenQuake Engine software provides calculation and assessment of seismic hazard, risk and decision-making tools via the data, methods and standards that are being developed by **<a href="http://www.globalquakemodel.org" target="_blank">GEM</a>** (Global Earthquake Model) and its collaborators.
+The OpenQuake Engine software provides calculation and assessment of seismic hazard, risk and decision-making tools via the data, methods and standards that are being developed by **[GEM](http://www.globalquakemodel.org)** (Global Earthquake Model) and its collaborators.
```

Comment:
Do we want to talk about `decision-making tools` ? I think we should write `social vulnerability` here.